subsys/net/ip/connection.c

```diff
@@ -280,6 +280,7 @@ int net_conn_register(uint16_t proto, uint8_t family,
 {
```

Comment:
Minor nit, but you should put the `Fixes #70020` to only one commit that has the actual fix. It looks like it is the 2nd one, all the other commits do not have to mention that.
So in the first commit, you can remove the
```
This is a prepared commit for fixes https://github.com/zephyrproject-rtos/zephyr/issues/70020.

Fixes: https://github.com/zephyrproject-rtos/zephyr/issues/70020
```
lines.

Reply:
Thank you. I will update the commit log.